apps/cdk/src/app/frontend-production.ts

```diff
@@ -0,0 +1,16 @@
+import { App } from 'aws-cdk-lib';
+
+import { waitForStackIdle } from '../lib/wait-for-stack-idle';
+import { FrontendStack } from '../stacks/frontend';
+
+async function main() {
+    const stackName = 'antalmanac-frontend-production';
+
+    await waitForStackIdle(stackName);
+
+    const app = new App({ autoSynth: true });
+
+    new FrontendStack(app, stackName, { env: { region: 'us-west-1' } });
```

Comment:
This stack was written for staging. You need to change it to support prod deployment. Also, check with @alexanderl19 about how this would affect DNS resolution, since it's been pointing at GithubPages.